ros2launch/ros2launch/command/launch.py

```diff
@@ -79,6 +79,9 @@ def add_arguments(self, parser, cli_name):
         parser.add_argument(
             '-d', '--debug', default=False, action='store_true',
             help='Put the launch system in debug mode, provides more verbose output.')
+        parser.add_argument(
+            '-f', '--log-file-name', default=None,
+            help='Name of the logging file')
```

Comment:
How about this, user can understand what needs to be set and how it is processed?

```suggestion
            help='Name of the log file (postfixed with .log automatically if not provided).')
```